lib/ret/media_search.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Ret.MediaSearchQuery do
+  @enforce_keys [:api]
+  defstruct [:api, :user]
+end
+
+defmodule Ret.MediaSearch do
+  import Ret.HttpUtils
+
+  def search(%Ret.MediaSearchQuery{api: "sketchfab", user: user}) do
+    with api_key when is_binary(api_key) <- resolver_config(:sketchfab_api_key) do
+      res =
+        "https://api.sketchfab.com/v3/search?type=models&downloadable=true&user=#{user}"
```

Comment:
I guess we're ignoring pagination and sorting? Picking a `sort_by` might be useful. The default "by relevance" is not very useful if you're just getting a user's model without an actual keyword query. Maybe by newest first?